test/e2e/gorillamux/traces.json

```diff
@@ -0,0 +1,52 @@
+{
+  "resourceSpans": [
+    {
+      "resource": {
+        "attributes": [
+          {
+            "key": "service.name",
+            "value": {
+              "stringValue": "sample-app"
+            }
+          },
+          {
+            "key": "telemetry.sdk.language",
+            "value": {
+              "stringValue": "go"
+            }
+          }
+        ]
+      },
+      "scopeSpans": [
+        {
+          "scope": {
+            "name": "github.com/gorilla/mux"
+          },
+          "spans": [
+            {
+              "attributes": [
+                {
+                  "key": "http.method",
+                  "value": {
+                    "stringValue": ""
```

Comment:
It also seems weird that these values are empty (not "Get" for example).

Reply:
https://github.com/open-telemetry/opentelemetry-go-instrumentation/pull/46#discussion_r1152142583